README.md

```diff
@@ -1,2 +1,88 @@
 # app.js
-GitHub Apps toolset for Node.js
+
+> Official GitHub Platform for GitHub Apps in Node.js
+
+## GitHub Apps
+
+`@octokit/app.js` allows to authenticate as GitHub Apps, installations, listen
+to webhooks, and respond to them using the GitHub API.
+
+### Authenticating as an App
+
+In order to authenticate as a GitHub App, you need to generate a Private Key
+and use it to sign a JSON Web Token (jwt) and encode it. 
+
+```js
+const App = require('@octokit/app.js')
+
+const app = new App({id, privateKey})
+const appAuth = await app.appAuth()
+
+// Example of using authetincated app to GET an individual installation
+// https://developer.github.com/v3/apps/#find-installations
```

Comment:
that should be https://developer.github.com/v3/apps/#find-repository-installation